ui/src/shared/components/CodeSnippet.tsx

```diff
@@ -28,18 +28,17 @@ class CodeSnippet extends PureComponent<Props> {
 
     return (
       <div className="code-snippet" data-testid={testID}>
-        <FancyScrollbar
+        <DapperScrollbars
           autoHide={false}
-          autoHeight={true}
-          maxHeight={400}
+          autoSizeHeight={true}
```

Comment:
should we set the maxHeight here?

Reply:
I can never decide the best way to handle these things. It's either set it inline using a style object (which I don't like cuz it's separate from the other styles), or create a class for it (which sometimes means making a stylesheet with a single class).

🤷‍♂